d3d9ex/AutoFix.cpp

```diff
@@ -172,12 +177,23 @@ HRESULT MainContext::CreateVertexBuffer(IDirect3DDevice9* pIDirect3DDevice9, UIN
 				//Pool = D3DPOOL_DEFAULT;
 				cinterface::VertexBufferFix(*ppVertexBuffer);
 			}
+
+
+			// TODO Move this somewhere else... 
+			// I've tried to move it to hkIDirect3D9::ApplyCreateDeviceFix but I got an incorrect ref count error message in the log (why?)
+			IDirect3DSwapChain9* swapChain = NULL;
+			HRESULT getSwapChainResult = pIDirect3DDevice9->GetSwapChain(0, &swapChain);
+			context.HookPresent(&swapChain);

```

Comment:
Any ideas about this?

Reply:
I'm back in home but i need to catch up with work. I will check this when i have some time.

Reply:
No problem, there's no rush. Glad to hear you are back home!